server/src/main/java/org/elasticsearch/search/aggregations/support/CoreValuesSourceType.java

```diff
@@ -416,6 +438,8 @@ public String typeName() {
         return value();
     }
 
+    public static final Logger log = LogManager.getLogger(CoreValuesSourceType.class);
```

Comment:
I think this belongs at the top of the file.

Reply:
This is an enum, not a class, so I can't put it at the top.  The enum constants need to be defined first.  I can put it a couple lines higher if you think that will be better.